src/gei/Commands/MigrateRepo/MigrateRepoCommand.cs

```diff
@@ -188,6 +189,8 @@ public override MigrateRepoCommandHandler BuildHandler(MigrateRepoCommandArgs ar
             var ghesVersionChecker = ghesVersionCheckerFactory.Create(ghesApi);
             var warningsCountLogger = sp.GetRequiredService<WarningsCountLogger>();
 
+            log.AddRedactionPattern(new Regex("\\?token=[A-Z0-9]{29}"));
```

Comment:
I have manually tested that this works as expected on GHES 3.7, and the URL is not logged.